### PR TITLE
 updated the detail issue #7 added a navbar link for contributor

### PR DESCRIPTION
**Pull request by chetnaSharma24**
hello
 I have added the contributor link on the home page , i hope now it can easily show the page.

currently I am working on footer i.e issue#4 so footer file has also been uploaded here.

I hope now it will work fine.

@anushbhatia kindly check

screencapture-localhost-3001-2020-08-17-02_48_13

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Nice work @chetnaSharma24 .
Please fix the merge conflict

**Comment by chetnaSharma24** [comment]
@smaranjitghose  
hey I have tried to resolve merge conflict  but i don't understand why all checks get failed  could you please brief about it.

**Comment by anushbhatia** [comment]
don't worry about that

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -2,8 +2,38 @@ import React from "react";
 import { NavLink } from "react-router-dom";
 import { FaPaperPlane } from "react-icons/fa";
 
+function Navbar() {
+    return(
+        <NavWrapper className="navbar navbar-expand-sm navbar-light px-sm-5">  
+            <div className="ml-auto navbar-nav">
+                <Link to="/">
+                    <div className="nav-item text-dark px-4 font-weight-bold">
```
Comment (abhu-A-J) [inline src/components/Navbar.jsx:10]: There are two navbar component on this page! The one below is well written and scalable to delete this one on the top.

File: src/components/Detail.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                </div>
+                <div className="row">
+                <div className="col-sm-12 col-md-6" align="center">
+                    <div className="card text-center">
+                    <img className="card-img-top"  src={boyimage}  alt="Card image cap"></img>
+                    <div className="card-body">
+                        <p className="card-text" ><b>Anush Bhatia</b></p>
+                        <p className="card-text label"><i>Designer</i></p>
+                    </div>
+                    </div>
```
Comment (abhu-A-J) [inline src/components/Detail.js:27]: We can make this into a seperate card component taking the details as props , it's repeated a lot of times.

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -2,8 +2,38 @@ import React from "react";
...
+        </NavWrapper>
+    )
+}
+
 const Navbar = () => {
 
+
   /*-------All Nav links goes here------------------ */
   const Navlinks = [
     { name: "Home", path: "/" },
```
Comment (abhu-A-J) [inline src/components/Navbar.jsx:39]: This Navbar is well Written and can be kept instead of the one of top . So delete the top one!

**Pull request merged**